### ISSUE TITLE
Does not detect the correct hidraw device

### ISSUE THREAD

**Issue by xcorat** [Lekensteyn/ltunify#4]
Trying to list devices gives an error:
```
$ ltunify list
write: Broken pipe
Failed to retrieve notification state
```

Seems like it's trying to access the wrong device, and I was wondering if there's a fix.  Specifying the device works,
```
$ ltunify -d /dev/hidraw3 list
Devices count: 1
Connected devices:
idx=1   Mouse   M185
$ ltunify -d /dev/hidraw2 list
write: Broken pipe
Failed to retrieve notification state
```

More info:
```
$ ls /dev/hidraw*
/dev/hidraw0  /dev/hidraw1  /dev/hidraw2  /dev/hidraw3

$ dmesg | grep hidraw
[54940.145922] hid-generic 0003:046D:C52F.0028: input,hidraw2: USB HID v1.11 Mouse [Logitech USB Receiver] on usb-0000:00:14.0-4/input0
[54940.205888] hid-generic 0003:046D:C52F.0029: input,hiddev97,hidraw3: USB HID v1.11 Device [Logitech USB Receiver] on usb-0000:00:14.0-4/input1
```

**Comment by Lekensteyn** [comment]
Can you include the kernel version in use and the output of `lsusb`?

**Comment by bgodard** [comment]
I have the same problem, but even with specifying hidraw1 I get:
```
Unknown protocol 0x0a in devcon notif
```
for both mouse and keyboard.

I have MK270 logitech mouse+keyboard combo.

```
$ lsusb 
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 003: ID 04f2:b65c Chicony Electronics Co., Ltd 
Bus 001 Device 005: ID 8087:0a2b Intel Corp. 
Bus 001 Device 006: ID 093a:2510 Pixart Imaging, Inc. Optical Mouse
Bus 001 Device 004: ID 046d:c534 Logitech, Inc. Unifying Receiver
Bus 001 Device 002: ID 05e3:0608 Genesys Logic, Inc. Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
$ uname -a
Linux venus 5.3.0-26-generic #28~18.04.1-Ubuntu SMP Wed Dec 18 16:40:14 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
$ sudo bin/ltunify list
write: Broken pipe
Failed to retrieve notification state
$ sudo bin/ltunify -d /dev/hidraw0 list
write: Broken pipe
Failed to retrieve notification state
$ sudo bin/ltunify -d /dev/hidraw1 list
Devices count: 2
Unknown protocol 0x0a in devcon notif
Unknown protocol 0x0a in devcon notif
Connected devices:
$ sudo bin/ltunify -d /dev/hidraw2 list
write: Broken pipe
Failed to retrieve notification state
$ sudo bin/ltunify -d /dev/hidraw3 list
Failed to retrieve notification state
$ sudo bin/ltunify -d /dev/hidraw4 list
Failed to retrieve notification state
```

**Comment by illtud** [comment]
I get same as bgodard. Logitec mk270, nano receiver, 5.6.12-300.fc32.x86_64. 

> $ dmesg | grep hidraw
[    1.417710] hid-generic 0003:413C:2005.0001: input,hidraw0: USB HID v1.10 Keyboard [DELL DELL USB Keyboard] on usb-0000:39:00.0-1/input0
[    2.706124] jabra 0003:0B0E:0422.0002: input,hiddev96,hidraw1: USB HID v1.11 Device [Jabra SPEAK 510 USB] on usb-0000:00:14.0-1.2/input3
[    2.828563] hid-generic 0018:04F3:24F0.0003: input,hidraw2: I2C HID v1.00 Device [ELAN24F0:00 04F3:24F0] on i2c-ELAN24F0:00
[    2.860668] hid-generic 0018:06CB:76AF.0004: input,hidraw3: I2C HID v1.00 Mouse [DELL07E6:00 06CB:76AF] on i2c-DELL07E6:00
[    2.978751] hid-multitouch 0018:04F3:24F0.0003: input,hidraw2: I2C HID v1.00 Device [ELAN24F0:00 04F3:24F0] on i2c-ELAN24F0:00
[    3.027129] hid-multitouch 0018:06CB:76AF.0004: input,hidraw3: I2C HID v1.00 Mouse [DELL07E6:00 06CB:76AF] on i2c-DELL07E6:00
[   65.598743] hid-generic 0003:046D:C534.0005: input,hidraw4: USB HID v1.11 Keyboard [Logitech USB Receiver] on usb-0000:00:14.0-1.4/input0
[   65.655050] hid-generic 0003:046D:C534.0006: input,hiddev97,hidraw5: USB HID v1.11 Mouse [Logitech USB Receiver] on usb-0000:00:14.0-1.4/input1
[   78.141553] logitech-djreceiver 0003:046D:C534.0005: hidraw4: USB HID v1.11 Keyboard [Logitech USB Receiver] on usb-0000:00:14.0-1.4/input0
[   78.252578] logitech-djreceiver 0003:046D:C534.0006: hiddev97,hidraw5: USB HID v1.11 Mouse [Logitech USB Receiver] on usb-0000:00:14.0-1.4/input1
[   78.307836] hid-generic 0003:046D:4023.0007: input,hidraw6: USB HID v1.11 Keyboard [Logitech Wireless Keyboard PID:4023] on usb-0000:00:14.0-1.4/input1:1
[   78.308959] hid-generic 0003:046D:4054.0008: input,hidraw7: USB HID v1.11 Mouse [Logitech Wireless Mouse PID:4054] on usb-0000:00:14.0-1.4/input1:2
[   79.834425] logitech-hidpp-device 0003:046D:4023.0007: input,hidraw6: USB HID v1.11 Keyboard [Logitech Wireless Keyboard PID:4023] on usb-0000:00:14.0-1.4/input1:1
[   79.859662] logitech-hidpp-device 0003:046D:4054.0008: input,hidraw7: USB HID v1.11 Mouse [Logitech Wireless Mouse] on usb-0000:00:14.0-1.4/input1:2
$ for i in {4..7} ; do echo "ltunify -d /dev/hidraw${i}" ; ltunify -d /dev/hidraw${i} list ; done
ltunify -d /dev/hidraw4
write: Broken pipe
Failed to retrieve notification state
ltunify -d /dev/hidraw5
Devices count: 2
Unknown protocol 0x0a in devcon notif
Unknown protocol 0x0a in devcon notif
Connected devices:
ltunify -d /dev/hidraw6
Failed to retrieve notification state
ltunify -d /dev/hidraw7
Failed to retrieve notification state

**Comment by Lekensteyn** [comment]
@bgodard @illtud Both of you have a particular Nano receiver model (046d:c534) which are quite limited in hardware. See also https://pwr-solaar.github.io/Solaar/capabilities

What are you trying to do? Pairing a new device? For this nano receiver, pairing a new device will replace the old one as documented on the previous link.

You can also try Solaar if you need more features.

**Comment by Lekensteyn** [comment]
Hi, I've just released ltunify v0.3 with support for the c534 Nano receiver. I've tested it with a MK270, please reopen if the problem persists. Thanks for reporting!
